### DISCUSSION TITLE
How to open two or more drawers on the same page

### DISCUSSION THREAD

**Discussion by romandubovyi** [ng-matero/extensions#198] [Q&A]
How to open two or more drawers on the same page, if I want to close the last drawer then the others stay open. For example, an open a second drawer from exist instance, not closing the first one.

I tried to use viewContainerRef in MtxDrawerConfig but it does not work.

**Comment by nzbin** [comment]
I'm sorry, it doesn't support multi drawers now.

**Comment by nzbin** [comment]
Please try the new version 18.1.0.
